### PR TITLE
first draft

### PR DESCRIPTION
**Pull request by danhellem**

### CODE REVIEW

File: docs/compare/Azure-Boards-vs-Jira.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+## Easily migrate from Jira Software
+
+Migrate from Jira Software to Azure DevOps in just a few minutes.
+Export from Jira Software to Microsoft Excel and then use the free
+[Excel add-in](https://docs.microsoft.com/azure/devops/boards/backlogs/office/track-work)
+to import into Azure Boards.
+
+You can also use free community tools such as Solidify's
+[Jira to Azure DevOps work item migration tool](https://marketplace.visualstudio.com/items?itemName=solidify-labs.jira-devops-migration)
+or perform high-fidelity migrations and even _synchronization_ using partner tools such as
```
Comment (martinwoodward) [inline docs/compare/Azure-Boards-vs-Jira.md]: Can we add a link to TaskTop in the sync section?

File: docs/compare/Azure-Boards-vs-Jira.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+| ------------------------------------ | :----------: | :-----------: | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Agile Process                        |      Y       |       Y       | Azure Boards supports basic, agile, scrum, and cmmi with the ability to switch between.                                                                                                                                                                                                                                                                                          |
```
Comment (martinwoodward) [inline docs/compare/Azure-Boards-vs-Jira.md]: Should we have something about integration to GitHub (including a link to the GitHub app for Boards)

File: docs/compare/Azure-Boards-vs-Jira.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
+---
+title: Azure Boards vs Jira
+description: Explanation of Azure Boards and how it compares to Atlassian Jira
+ms.prod: devops
+ms.technology: devops-learn
+ms.topic: conceptual
+ms.manager: douge
+ms.author: davemcki
```
Comment (martinwoodward) [inline docs/compare/Azure-Boards-vs-Jira.md]: Feel free to put yourself in here

File: docs/compare/Azure-Boards-vs-Jira.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+ms.date: 3/11/2019
+---
+
+# Azure Boards vs Jira Software
+
+Azure DevOps is a cloud-based, open, and integrated platform for enterprise-scale DevOps. Azure Boards is the specific Azure DevOps service focused on agile project management.
+
+To compare to Jira Software, try Azure Boards for free and get:
```
Comment (martinwoodward) [inline docs/compare/Azure-Boards-vs-Jira.md]: Maybe just

[try Azure Boards for free](https://azure.microsoft.com/services/devops/boards) here and then get rid of the sign-up link below?

File: docs/compare/Azure-Boards-vs-Jira.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+
+- Access for up to five users
+- Access for an unlimited number of stakeholders
+- Dashboards, portfolio management and other Jira add-ons included
+- Access to other Azure DevOps services
+  (e.g., [Azure Repos](https://azure.microsoft.com/services/devops/repos),
+  [Azure Pipelines](https://azure.com/pipelines), etc.)
+
+[Sign-up here](https://azure.microsoft.com/services/devops/boards) and compare for yourself _for FREE_!
```
Comment (martinwoodward) [inline docs/compare/Azure-Boards-vs-Jira.md]: See previous comment.  Include the link earlier and then remove his - feels a bit sales-y for docs

File: docs/compare/Azure-Boards-vs-Jira.md [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+ms.technology: devops-learn
+ms.topic: conceptual
+ms.manager: douge
+ms.author: davemcki
+ms.date: 3/11/2019
+---
+
+# Azure Boards vs Jira Software
+
+Azure DevOps is a cloud-based, open, and integrated platform for enterprise-scale DevOps. Azure Boards is the specific Azure DevOps service focused on agile project management.
```
Comment (dmckinstry) [inline docs/compare/Azure-Boards-vs-Jira.md]: Following the lead of "Azure has DevOps baked in" consider removing "DevOps" in the second sentence...  Azure Boards is the specific *Azure service*..."

File: docs/compare/Azure-Boards-vs-Jira.md (hunk 5 above)
Comment (dmckinstry) [reply docs/compare/Azure-Boards-vs-Jira.md]: I agree.  We've come a long way from the original requirements.  When this was in the early stages it was all about putting marketing links in... happy that we're pulling them out (or making them less "sales-y"

File: docs/compare/Azure-Boards-vs-Jira.md (hunk 1 above)
Comment (dmckinstry) [reply docs/compare/Azure-Boards-vs-Jira.md]: TaskTop? OpsHub? Other?  When I started this, Ewald was all about TFS4Jira but we have a few integrations stories (including community).  How many do we want to drive?

**Comment by martinwoodward** [comment]
I'm fine with just adding TaskTop for sync.

**Pull request merged**